projects/components/dialog-wrapper/src/dialog-wrapper.component.scss

```diff
@@ -0,0 +1,27 @@
+.ps-dialog-wrapper__action-bar-container {
+  display: flex;
+  width: 100%;
+}
+
+.ps-dialog-wrapper__error {
+  color: var(--ps-error);
+}
+
+.ps-dialog-wrapper__buttons {
+  margin-left: auto;
+}
+
+ps-dialog-wrapper {
+  display: block;
+  /* Breiter geht bei einem MatDialog nicht, ohne das ein horrizontaler Scrollbalken kommt. */
```

Comment:
Please translate your comments to English

Reply:
I've removed the other comments since they were a little specific to our product. The comment for setting the width via style on the component should be moved to the docs when they are reworked.